app/models/authorization.rb

```diff
@@ -16,6 +16,17 @@ def id
     default_account.id
   end
 
+  def feeder_ui_access
+    if (token.resources(:feeder, :read_private) - token.resources(:cms, :read_private)).any?
```

Comment:
Based on my understanding of what this is doing, `token.resources(:feeder, :read_private).any? && token.resources(:cms, :read_private).empty?` may be more obvious, but I think it does that same thing.

Reply:
Ah true, since in `roles.yml` ... every role that has `cms:read-private` also has `feeder:read-private`.